action.yml

```diff
@@ -14,6 +14,9 @@ inputs:
       ideas.
     default: stable-2.11
     required: true
+  docker-image:
+    description: Docker image used by ansible-test
```

Comment:
Is it possible to also specify an input `type:` field too? I've heard it's supported now.

Reply:
yes it is possible. If you agree I suggest to do that in a next pass to cover all inputs at once.

Reply:
I guess it could be postponed.